### PR TITLE
general custom content consistency pass

### PR DESCRIPTION
**Pull request by Darkmajia**
went through a bunch of custom content from impstation and other server ports to clean up inconsistent text and stuff. did i test this pr in-game? no. its just yaml

**Changelog**
- tweak: A few item names and other pieces of text have been adjusted for consistency.

### CODE REVIEW

File: Resources/Locale/en-US/_Goobstation/guidebook/ghost-gui.ftl [hunk 1]
```diff
@@ -2,10 +2,10 @@ ghost-target-window-ghostbar = Ghost Bar
...
-    YOU CAN NOT BE REVIVED INTO YOUR OLD BODY OR BRAIN ONCE YOU VISIT THE GHOST BAR.
+    YOU CANNOT BE REVIVED INTO YOUR OLD BODY OR BRAIN ONCE YOU VISIT THE GHOST BAR.
 
-    Treat this role, and station, as you would just being a regular Ghost.
+    Treat this role, and station, as you would just being a regular ghost.
 
     You may talk about all current round events without the need for LOOC, and you remember everything from your previous life.
 
-    DO NOT attack others, start fights, or attempt to break the station. If you see anyone doing this, please Ahelp and they will be promptly thrown into space. Also, if you decide to leave the bar, you DO NOT remember anything from being here, or your life before it.
+    DO NOT attack others, start fights, or attempt to break the station. If you see anyone doing this, please ahelp and they will be promptly thrown into space. Also, if you decide to leave the bar, you DO NOT remember anything from being here, or your life before it.
```
Comment (hivehum) [inline Resources/Locale/en-US/_Goobstation/guidebook/ghost-gui.ftl:11]: pretty sure the game refers to it as AHelp elsewhere? rules doc sure does

File: Resources/Prototypes/_Impstation/Entities/Objects/Misc/books_author.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@
 - type: entity
   parent: BookBase
   id: BookGatsby
-  name: The Great Gatsby by F. Scott Fitzgerald
+  name: the great gatsby by F. Scott Fitzgerald
```
Comment (hivehum) [inline Resources/Prototypes/_Impstation/Entities/Objects/Misc/books_author.yml:61]: this should be capitalized, i think?

Comment (Darkmajia) [reply Resources/Prototypes/_Impstation/Entities/Objects/Misc/books_author.yml:61]: proooobably, i was just going off the other books being uncapitalised

Comment (hivehum) [reply Resources/Prototypes/_Impstation/Entities/Objects/Misc/books_author.yml:61]: all books should have capitalized titles. they r proper nowns

File: Resources/Locale/en-US/_Impstation/popup/snail.ftl [hunk 3]
```diff
@@ -1 +1 @@
-snail-hurt-by-chem-popup = Ouch! Thats not snail friendly.
\ No newline at end of file
+snail-hurt-by-chem-popup = Ouch! That's not snail friendly.
```
Comment (irismessage) [inline Resources/Locale/en-US/_Impstation/popup/snail.ftl:1]: missing newline

File: Resources/Locale/en-US/_Impstation/reagents/meta/consumable/drink/alchohol.ftl [hunk 4]
```diff
@@ -37,7 +37,7 @@ reagent-desc-bloodier-mary = AH, AH, AHH!
 reagent-name-salvagerssoothe = salvager's soothe
 reagent-desc-salvagerssoothe = Come home safe, okay? We're counting on you.
 
-reagent-name-donttelldsheriff = don't tell D sherrif
+reagent-name-donttelldsheriff = don't tell D sheriff
```
Comment (irismessage) [inline Resources/Locale/en-US/_Impstation/reagents/meta/consumable/drink/alchohol.ftl:40]: was this typo intentional? idk if it's a reference to something. although the key is spelt sheriff

**Review by irismessage — APPROVED** [review]

**Comment by irismessage** [comment]
also how the fuck does the linter think this pr added a duplicate of camel soup

**Comment by Darkmajia** [comment]
i dont know and its fucking me up

File: Resources/Locale/en-US/_Impstation/reagents/meta/consumable/drink/alchohol.ftl (hunk 4 above)
Comment (Darkmajia) [reply Resources/Locale/en-US/_Impstation/reagents/meta/consumable/drink/alchohol.ftl:40]: pretty sure it wasnt, searching the discord for "d sheriff" came up with the original suggestion https://discord.com/channels/716470896985899219/1277112023158886481/1278355436206161983

**Comment by Darkmajia** [comment]
ran yaml linter locally and it passed with no errors so i'm not sure why it's running into them on github

**Comment by hivehum** [comment]
idk why github is allergic to my soup

**Pull request merged**